addons/sys_radio/stringtable.xml

```diff
@@ -3,26 +3,33 @@
     <Package name="sys_radio">
         <Key ID="STR_ACRE_sys_radio_radioSupplyCrate">
             <English>[ACRE] Radio Supply Crate</English>
+            <German>[ACRE] Funkgerätekiste</German>
             <Japanese>[ACRE] 無線機物資箱</Japanese>
             <Russian>[ACRE] Ящик с рациями</Russian>
         </Key>
         <Key ID="STR_ACRE_sys_radio_noGuiExternal">
             <English>Radio used externally. Only owner may configure.</English>
+            <German>Das Funkgerät wird extern benutzt. Nur der Besitzer kann es einstellen.</German>
         </Key>
         <Key ID="STR_ACRE_sys_radio_noGuiTurnedOut">
             <English>Radio rack not configurable while turned out.</English>
+            <German>Das Funkgerät kann nicht Oberluke verstellt werden</German>
```

Comment:
To avoid confusion, we could state that the radio rack can't be reached from this position / while turned out. Should change that in the English version as well.